packages/nextjs-mf/src/loaders/delegateLoader.ts

```diff
@@ -1,15 +1,24 @@
 import type { LoaderContext } from 'webpack';
+
 /**
+ * This function patches the default shared loader.
+ * It requires either the default delegate module or a custom one.
  *
- * Requires either the default delegate module or a custom one
- *
+ * @param {LoaderContext<Record<string, unknown>>} this - The loader context.
+ * @param {string} content - The content to be processed.
+ * @returns {string} The processed content with required delegates.
  */
 export default function patchDefaultSharedLoader(
   this: LoaderContext<Record<string, unknown>>,
   content: string
 ) {
```

Comment:
The logic inside the map function can be simplified by using a ternary operator. This can improve readability and maintainability.